sympy/controlsys/lti.py

```diff
@@ -0,0 +1,1363 @@
+
+        Examples
+        ========
+
+        >>> from sympy import Matrix, symbols
+        >>> from sympy.controlsys.lti import StateSpaceModel
+
+        The controllability matrix only depends on A and B:
+
+        >>> a0, a1, a2 = symbols('a:3')
+        >>> A = Matrix([[a0, a1, a2], [1, 0, 0], [0, 1, 0]])
+        >>> B = Matrix([1, 0, 0])
+        >>> StateSpaceModel(A, B).controllability_matrix()
+        Matrix([
+        [1, a0, a0**2 + a1],
+        [0,  1,         a0],
+        [0,  0,          1]])
+
+        """
+        res = self.represent[1]
+        for i in range(self.represent[0].shape[0] - 1):
+            res = res.row_join(self.represent[0] ** (i + 1) * self.represent[1])
+        return res
+
+    def controllable_subspace(self):
+        """ Returns a list of vectors that span the controllable subspace of the system.
+
+        This subspace consists of the states x0 for which there exists an input u : [t0, t1] -> R^k, that
+        transfers the state x(t0) = x0 to x(t1) = 0.
+
+        The controllable subspace of an lti system is equal to the image of its controllability matrix.
+
+        Examples
+        ========
+
+        >>> from sympy import Matrix, symbols
+        >>> from sympy.controlsys.lti import StateSpaceModel
+
+        The controllable subspace only depends on A and B:
+
+        >>> a0, a1, a2 = symbols('a:3')
+        >>> A = Matrix([[a0, a1, a2], [1, 0, 0], [0, 1, 0]])
+        >>> B = Matrix([1, 0, 0])
+        >>> StateSpaceModel(A, B).controllable_subspace()
+        [Matrix([
+        [1],
+        [0],
+        [0]]), Matrix([
+        [a0],
+        [ 1],
+        [ 0]]), Matrix([
+        [a0**2 + a1],
+        [        a0],
+        [         1]])]
+
+        """
+        return self.controllability_matrix().columnspace()
+
+    def is_controllable(self):
+        """ Returns True, if the system is controllable.
+
+        A lti system is called 'controllable' if the controllable subspace of the system equals the
+        whole state space R^n. This means, that every state x0 can be transfered to zero at any time.
+
+        The package implements the Eigenvector test for controllability
+
+        Examples
+        ========
+
+        >>> from sympy import Matrix, symbols
+        >>> from sympy.controlsys.lti import StateSpaceModel
+
+        The controllability only depends on A and B:
+
+        >>> a1, a2, b1, b2 = symbols('a1:3, b1:3')
+        >>> A = Matrix([[a1, 0], [0, a2]])
+        >>> B = Matrix([b1, b2])
+        >>> StateSpaceModel(A, B).is_controllable()
+        True
+
+        >>> StateSpaceModel(A, B.subs(b2, 0)).is_controllable()
+        False
+
+        """
+        for eigenvect_of_A_tr in self.represent[0].transpose().eigenvects():
+            for idx in range(eigenvect_of_A_tr[1]):
+                if (self.represent[1].transpose() * eigenvect_of_A_tr[2][idx]).is_zero:
+                    return False
+
+        return True
+
+    def cascade(self, other):
```

Comment:
I feel like "series" is the word that is more commonly used. Series and parallel connections in block diagrams.